README.md

```diff
@@ -99,7 +99,8 @@ docker run --env-file .env -p 5000:5000 todo.app:prod
 docker run --env-file .env -p 5000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/srv/todo_app todo.app:dev
 ```
 ``` bash
-docker run --env-file .env --mount type=bind,source="$(pwd)"/todo_app,target=/srv/todo_app todo.app:test
+docker run --env-file .env.test --mount type=bind,source="$(pwd)"/todo_app,target=/srv/todo_app todo.app:test tests
+docker run --env-file .env --mount type=bind,source="$(pwd)"/todo_app,target=/srv/todo_app todo.app:test tests_e2e
 ```
 
 To launch the development app easier you can use docker-compose command:
```

Comment:
It would be good to update this with the latest changes, e.g. the extra environment variables (also in `.env.template`) and the extra steps required in setting up an OAuth app with Github

Reply:
Completely forgot about the deployment env variables. Also forgot to create a new OAuth app as the existing app settings are just for local dev for now.

Reply:
Had to also change the redirect_uri to use `request.url_root` instead of the hardcoded 'localhost:5000'